src/bin/add_historical_data.rs

```diff
@@ -0,0 +1,527 @@
+            ),
+        ),
+        (
+            "mooglemod",
+            NewPlayer::new(
+                "mooglemod",
+                "267398828770983957",
+                Some("mooglemod#8456"),
+                Some("mooglemod"),
+                Some("xldAMBl417BaOP57"),
+            ),
+        ),
+        // (
+        //     "shkoople",
+        //     NewPlayer::new(
+        //         "shkoople",
+        //         "98548933424332800",
+        //         Some("shkoople#4144"),
+        //         Some("shkoople"),
+        //         Some("LNY0OkW1DZoKalP1"),
+        //     ),
+        // ),
+        // (
+        //     "buane",
+        //     NewPlayer::new(
+        //         "Buane",
+        //         "124664588485656582",
+        //         Some("Buane#5757"),
+        //         Some("buane"),
+        //         Some("41jgrbWPz3e7P5QE"),
+        //     ),
+        // ),
+        (
+            "shadyforce",
+            NewPlayer::new(
+                "shadyforce",
+                "379655195350794250",
+                Some("shadyforce"),
+                Some("shadyforcegames"),
+                Some("NJrM6PoY5oRdm5v2"),
+            ),
+        ),
+        (
+            "mcmonkey",
+            NewPlayer::new(
+                "McMonkey",
+                "178293242045923329",
+                Some("McMonkey#7533"),
+                Some("mcmonkey819"),
+                Some("AEk8wpokxZW5KQyV"),
+            ),
+        ),
+        (
+            "parisianplayer",
+            NewPlayer::new(
+                "parisianplayer",
+                "153404520259387393",
+                Some("parisianplayer#2994"),
+                Some("parisianplayer"),
+                Some("8QGZrB2MP0WNgk4V"),
+            ),
+        ),
+        (
+            "daaanty",
+            NewPlayer::new(
+                "daaanty",
+                "98625136076279808",
+                Some("daaanty#0264"),
+                Some("daaanty"),
+                Some("yDMLq1oZDEBOeQG8"),
+            ),
+        ),
+        (
+            "aurorasnerd",
+            NewPlayer::new(
+                "AuroraSnerd",
+                "724797547637506090",
+                Some("AuroraSnerd#0011"),
+                Some("aurorasnerd"),
+                Some("MqzQPW4jNK31L2R5"),
+            ),
+        ),
+    ]
+    .into_iter()
+    .map(|(k, v)| (k.to_string(), v))
+    .collect::<_>();
+    let mut db = raw_diesel_cxn_from_env()?;
+
+    let orm_players = get_all_players(&mut db)?;
+    validate_players(&missing_players, &orm_players)?;
+
+    for p in missing_players.into_values() {
+        if let Err(e) = p.save(&mut db) {
+            return Err(anyhow!("Error saving {p:?}: {e}"));
+        }
+    }
+
+    let orm_players = get_all_players(&mut db)?;
+    let s2 = Season::get_by_ordinal(2, &mut db)?;
+    let s4 = Season::get_by_ordinal(4, &mut db)?;
+
+    actually_make_season(s1, &orm_players, &mut db)?;
```

Comment:
i think with how `make_bracket` worked out, it would be nicer to have this as just making the `s1` object directly and using `make_bracket` on it, but im not sure i care about refactoring much since this is a one-off script